### ISSUE TITLE
CMake policy OLD setting deprecated

### ISSUE THREAD

**Issue by kartikkumar** [SRombauts/SQLiteCpp#184]
When I build the latest version, CMake reports the following:

```
CMake Deprecation Warning at CMakeLists.txt:8 (cmake_policy):
  The OLD behavior for policy CMP0048 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.
```

I'm not familiar with `cmake-policies` so I figured I'd report this here, in case it hasn't been noted yet as something to address.

**Comment by SRombauts** [comment]
Ok, thanks, I think this is the consequence or a very recent pull request I merged

**Comment by SRombauts** [comment]
Fixed in commit 804879d2 by reverting part of the recent #170
